lambdas/service/app.py

```diff
@@ -471,19 +471,27 @@ def openapi():
 }
 
 fast_health_keys = {
-    **{
-        prop.key: format_description(prop.description)
-        for prop in HealthController.fast_properties['service']
-    },
-    **health_up_key
+    p: d for p, d in sorted(
```

Comment:
If changing `HealthController.fast_properties` from `Mapping[str, Set[health_property]]` to `Mapping[str, Tuple[health_property]]` solves the inconsistent ordering, I think that would be a more elegant solution.

Reply:
changing from set to a tuple does not resolve the non-deterministic ordering. :|

Reply:
https://github.com/DataBiosphere/azul/pull/2997#discussion_r625339700